### PR TITLE
Update draft-js version to 0.11.4

### LINKED ISSUE #292: Update draft.js to v0.11.0
The release features a number of bugfixes and enhancements. Consider the upgrade.

### PR DESCRIPTION
**Pull request by lousander**
## Related Issue
Closes #292 

## Proposed Changes
 - Update draft js version

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3068,10 +3068,10 @@ core-js-compat@^3.6.0:
     browserslist "^4.8.3"
     semver "7.0.0"
 
-core-js@2, core-js@^2.4.0, core-js@^2.6.5:
+core-js@2, core-js@^2.4.0, core-js@^2.4.1, core-js@^2.6.5:
   version "2.6.11"
-  resolved "https://registry.yarnpkg.com/core-js/-/core-js-2.6.11.tgz#38831469f9922bded8ee21c9dc46985e0399308c"
-  integrity sha512-5wjnpaT/3dV+XB4borEsnAYQchn00XSgTAWKDkEqv+K8KevjbzmofK6hfJ9TZIlpj2N0xQpazy7PiRQiWHqzWg==
+  resolved "https://artifactory.globoi.com/artifactory/api/npm/npm-repos/core-js/-/core-js-2.6.11.tgz#38831469f9922bded8ee21c9dc46985e0399308c"
```
Comment (marcelometal) [inline yarn.lock]: Hi!  "artifactory.globoi.com" is not valid, could you please fix it?

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "backstage-modal": "^0.3.2",
     "classnames": "^2.2.5",
-    "draft-js": "0.10.5",
+    "draft-js": "0.11.0",
```
Comment (marcelometal) [inline package.json]: Why not 0.11.4?

File: yarn.lock (hunk 1 above)
Comment (lousander) [reply yarn.lock]: Done

File: package.json (hunk 2 above)
Comment (lousander) [reply package.json]: Done. Didn't want to jump too many versions but tested with 0.11.4 and seems ok.

**Pull request merged**